### PR TITLE
[WIP] Dialog editor entry point treeview

### PR DESCRIPTION
**Pull request by Hyperkid123**
This is a component that enables to choose entry point from a tree view. 
At this point it uses the older jQuery treeview. The goal is to make treeview proper angular component. It should be a dumb component, but since it uses data from tree builder, and lazy loading, it might be practical to keep data loading inside of this component. 

files:
/src/dialog-tree

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@miq-bot add_label wip

**Comment by Hyperkid123** [comment]
@miq-bot assign martinpovolny

**Comment by Hyperkid123** [comment]
@miq-bot remove_label wip

File: demo/services/availableComponentsService.ts [hunk 1]
```diff
@@ -93,6 +93,12 @@ export default class AvailableComponentsService {
           'TreeView',
           '/tree-view',
           require('./../views/tree-view/basic.html'), 'demoTreeView as vm')
+      ]),
+      new AvailableGroup('dialog-user', 'Dialog-User Component', '/dialog-user', [
+        new AvailableComponent('dialog-user',
+          'Dialog-user',
+          '/dialog-user',
+          require('./../views/dialog/user.html'), 'demoDialogs as vm')
```
Comment (skateman) [inline demo/services/availableComponentsService.ts]: You don't need this, it's already there on lines 86-89

**Comment by skateman** [comment]
@himdel @karelhala some new files appeared in the `dist` folder, is that OK here?

File: demo/services/availableComponentsService.ts (hunk 1 above)
Comment (Hyperkid123) [inline demo/services/availableComponentsService.ts]: deleted :+1:

**Comment by karelhala** [comment]
@skateman that's not ok. Those are fonts which should not be pushed to git. However there is missing rule in `.gitignore`

**Comment by skateman** [comment]
@Hyperkid123 let's extract the `dialog-tree` into a more generic `tree-selector` component with a `selectables` regexp attribute that can specify what can be selected.

**Comment by Hyperkid123** [comment]
@miq-bot add_label wip

**Comment by himdel** [comment]
gitignoring fonts solved in https://github.com/ManageIQ/ui-components/pull/126 :)

**Pull request closed without merging**